apps/sms/views/shared/js/activity_handler.js

```diff
@@ -3,7 +3,10 @@
 
 /*global Utils, MessageManager, Compose, NotificationHelper,
          Attachment, Notify, SilentSms, Threads, SMIL, Contacts,
-         ConversationView, Notification, Settings, Navigation */
+         ConversationView, Notification, Settings, Navigation,
+         ActivityClient,
+         ActivityShim
```

Comment:
Oops, you're right, I compared "h" and "l" but forgot the "C" and "S" that are before. Forget me.